extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -463,22 +488,33 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     // Not supported.
   }
 
-  /** @override */
-  fullscreenEnter() {
+   /** @override */
+   fullscreenEnter() {
```

Comment:
need to fix lint issues.

`amp lint --fix --local_files` should do the trick. (make sure you have committed all your changes first before running the command since it is destructive)